src/Service/DigitalPostHelper.php

```diff
@@ -72,6 +77,106 @@ public function sendDigitalPost(string $cpr, string $name, Address $address, str
+        // but we add it here and modify it later if necessary
+        $currentDigitalPost->setSubject($subject);
+
+        $this->entityManager->persist($currentDigitalPost);
+
+        if (count($digitalPostAttachments) > 0) {
+            foreach ($digitalPostAttachments as $attachment) {
+                assert($attachment instanceof DigitalPostAttachment);
+                $attachmentSize = filesize($this->getFilePath($attachment->getDocument()));
+                // Attachments are made from documents i.e. we are guaranteed they are less than 80mb
+
+                // Check if adding attachment would violate restrictions
+                if ($currentSizeOfDigitalPost + $attachmentSize >= $sizeLimit || $currentNumberOfAttachments >= $attachmentLimit) {
+                    // Persist current digital post entity and make new for current attachment
+                    $digitalPosts[] = $currentDigitalPost;
+
+                    $currentDigitalPost = new DigitalPostEntity();
+
+                    // Handle non-attachment properties on DigitalPost
+                    foreach ($digitalPostRecipients as $recipient) {
+                        $currentDigitalPost->addRecipient($recipient);
+                    }
+
+                    $currentDigitalPost->setDocument($document);
+                    $currentDigitalPost->setEntityType($entityType);
+                    $currentDigitalPost->setEntityId($entityId);
+                    $currentDigitalPost->setSubject($subject);
+
+                    $this->entityManager->persist($currentDigitalPost);
+                    $this->entityManager->flush();
+
+                    $currentSizeOfDigitalPost = $mainDocumentSize;
+                    $currentNumberOfAttachments = 0;
+                }
+
+                // Add current attachment
+                $attachment->setPosition($currentNumberOfAttachments + 1);
+                $currentDigitalPost->addAttachment($attachment);
+                $this->entityManager->persist($attachment);
+
+                $currentSizeOfDigitalPost += $attachmentSize;
+                ++$currentNumberOfAttachments;
+
+                $this->entityManager->flush();
+            }
+
+            $digitalPosts[] = $currentDigitalPost;
+
+            // Modify subject and pointers of digital post if more than one was created
+            $numberOfDigitalPosts = count($digitalPosts);
+
+            $previous = null;
+            if ($numberOfDigitalPosts > 1) {
+                $counter = 1;
+                foreach ($digitalPosts as $digitalPost) {
+                    assert($digitalPost instanceof DigitalPostEntity);
+                    if ($previous) {
+                        $digitalPost->setPrevious($previous);
+                        $previous->setNext($digitalPost);
+                    }
+                    $newSubject = $digitalPost->getSubject().' '.$counter.'/'.$numberOfDigitalPosts;
```

Comment:
Consider using `sprintf('%s (%d/%d)', …)` here.

Reply:
Used `sprintf` to generate new subject.